prisma/schema.prisma

```diff
@@ -32,6 +32,8 @@ model EventType {
   periodCountCalendarDays Boolean?
   requiresConfirmation Boolean @default(false)
   minimumBookingNotice Int @default(120)
+
+  @@unique([userId, slug])
```

Comment:
⚠️ ⚠️ 

Potentially volatile migration if there are any users who created 2 event types with the same slug.

To check:

```sql
select "userId", "slug", count(*)
from "EventType"
group by "userId", "slug"
HAVING count(*) > 1
```